src/components/card/card.stories.tsx

```diff
@@ -460,151 +519,165 @@ DifferentCardFooterPadding.parameters = {
-            ml={2}
-          >
-            Edit Button
-          </Button>
-          <VerticalDivider tint={80} py={0} px={2} h={30} />
-          <Button
-            p={0}
-            iconPosition="after"
-            iconType="edit"
-            buttonType="tertiary"
-            ml={2}
-          >
-            Edit Button
-          </Button>
-        </Box>
-      </CardFooter>
     </Card>
-    <Card cardWidth="400px">
+    <Card
+      width="400px"
+      footer={
+        <CardFooter variant="transparent" px={2} py={1}>
+          <Box
+            width="100%"
+            alignItems="center"
+            display="flex"
+            justifyContent="center"
+          >
+            <Button
+              p={0}
+              iconPosition="after"
+              iconType="edit"
+              buttonType="tertiary"
+            >
+              Edit Button
+            </Button>
+            <VerticalDivider tint={80} py={0} px={2} h={30} />
+            <Button
+              p={0}
+              iconPosition="after"
+              iconType="edit"
+              buttonType="tertiary"
+              ml={2}
+            >
+              Edit Button
+            </Button>
+            <VerticalDivider tint={80} py={0} px={2} h={30} />
+            <Button
+              p={0}
+              iconPosition="after"
+              iconType="edit"
+              buttonType="tertiary"
+              ml={2}
+            >
+              Edit Button
+            </Button>
+          </Box>
+        </CardFooter>
+      }
+    >
       <CardRow>
         <CardColumn>
           <Typography fontSize="16px" mt={2} mb={0} fontWeight="bold">
             Here is some text
           </Typography>
         </CardColumn>
       </CardRow>
-      <CardFooter variant="transparent" px={2} py={1}>
-        <Box
-          width="100%"
-          alignItems="center"
-          display="flex"
-          justifyContent="center"
-        >
-          <Button
-            p={0}
-            iconPosition="after"
-            iconType="edit"
-            buttonType="tertiary"
-          >
-            Edit Button
-          </Button>
-          <VerticalDivider tint={80} py={0} px={2} h={30} />
-          <Button
-            p={0}
-            iconPosition="after"
-            iconType="edit"
-            buttonType="tertiary"
-            ml={2}
-          >
-            Edit Button
-          </Button>
-          <VerticalDivider tint={80} py={0} px={2} h={30} />
-          <Button
-            p={0}
-            iconPosition="after"
-            iconType="edit"
-            buttonType="tertiary"
-            ml={2}
-          >
-            Edit Button
-          </Button>
-        </Box>
-      </CardFooter>
     </Card>
-    <Card>
+    <Card
+      footer={
+        <CardFooter p={2}>
+          <Box display="flex" width="100%" justifyContent="center">
+            <Link icon="link" href="https://carbon.sage.com/">
+              View Stripe Dashboard
```

Comment:
I agree these examples need refactoring, I intentionally only touched what I needed to as I want a full review of our stories to be done before I go through and make changes that might contradict its findings etc. I also want to avoid the mass conflicts it will cause for the storybook upgrade work